packages/gatsby/cache-dir/slice.js

```diff
@@ -32,6 +32,22 @@ export function Slice(props) {
     } else if (slicesContext.renderEnvironment === `engines`) {
       // if we're in SSR, we'll just render the component as is
       return <InlineSlice {...internalProps} />
+    } else if (slicesContext.renderEnvironment === `slices`) {
+      // we are not yet supporting nested slices
+
+      let additionalContextMessage = ``
+
+      // just in case generating additional contextual information fails, we still want the base message to show
+      // and not show another cryptic error message
+      try {
+        additionalContextMessage = `\n\nSlice component "${slicesContext.sliceRoot.name}" (${slicesContext.sliceRoot.componentPath}) tried to render <Slice id="${props.id}"/>`
```

Comment:
```suggestion
        additionalContextMessage = `\n\nSlice component "${slicesContext.sliceRoot.name}" (${slicesContext.sliceRoot.componentPath}) tried to render <Slice alias="${props.alias}"/>`
```

I think it'd be better to use alias here, as that's the prop that'll be used in their code.